fullstack-cert/react-projects/live-search/styles.css

```diff
@@ -0,0 +1,35 @@
+body {
+  font-family: Arial, sans-serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  background-color: #f4f4f4;
+}
+
+#search-container {
+  text-align: center;
+  background: white;
+  padding: 20px;
+  border-radius: 10px;
+  box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
+}
+
+#search-input {
+  padding: 10px;
+  width: 80%;
+  font-size: 16px;
```

Comment:
in general we want to avoid pixels for font sizes. But this can be removed since 16 is the default anyway. 

```suggestion
```